### PR TITLE
Fix mysql sku values

### PR DESCRIPTION
**Pull request by zikalino**
Gen4 values are not supported/valid any more.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
         "createMode": "Default"
       },
       "sku": {
-        "name": "B_Gen4_2",
+        "name": "B_Gen5_2",
```
Comment (praries880) [inline specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json]: Are examples the only places these values appear at? Is there associated documentation or swagger spec that needs to be updated as well?

Comment (zikalino) [reply specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json]: @praries880 I think examples is the only place where you can find these values in the REST API specs. If examples are wrong the only way to figure out proper values is to go to the portal, or use Azure CLI to create the server and then see what values are correct.

Anyway, thanks for reminding me. I did a search and found out that we could also update response values -- this PR only fixes values in the requests.

Comment (praries880) [reply specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json]: @zikalino will you be updating the PR to fix the values in the responses as well?

Comment (zikalino) [reply specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json]: @praries880 everything is already correct, I haven't found any occurrences of Gen4 anywhere. So I think we can go ahead and merge this PR.

**Review by praries880 — APPROVED** [review]

**Pull request merged**